plugins/source/gandi/codegen/recipes/certificates.go

```diff
@@ -0,0 +1,35 @@
+package recipes
+
+import (
+	"github.com/cloudquery/plugin-sdk/codegen"
+	"github.com/go-gandi/go-gandi/certificate"
+)
+
+func CertificateResources() []Resource {
+	return []Resource{
+		{
+			DataStruct:   &certificate.CertificateType{},
+			PKColumns:    []string{"id"},
+			ExtraColumns: []codegen.ColumnDefinition{SharingIDColumn},
+			TableName:    "gandi_certificates",
+
+			Template:         "resource_manual",
```

Comment:
Nit: I think we can call this template `resource` and have it be the default everywhere, since there are no other options right now anyway.